backend/private-graph/graph/schema.resolvers.go

```diff
@@ -5500,6 +5500,57 @@ func (r *queryResolver) SessionsHistogramClickhouse(ctx context.Context, project
 	}, nil
 }
 
+// SessionsReport is the resolver for the sessions_report field.
+func (r *queryResolver) SessionsReport(ctx context.Context, projectID int, query modelInputs.ClickhouseQuery) ([]*modelInputs.SessionsReportRow, error) {
+	project, err := r.isAdminInProjectOrDemoProject(ctx, projectID)
+	if err != nil {
+		return nil, err
+	}
+	workspace, err := r.GetWorkspace(project.WorkspaceID)
+	if err != nil {
+		return nil, err
+	}
+	retentionDate := GetRetentionDate(workspace.RetentionPeriod)
+
+	// If there's no admin for the context, use `admin=nil`
+	// (admin is used by the "viewed by me" filter)
+	admin, err := r.getCurrentAdmin(ctx)
+	if errors.Is(err, AuthenticationError) {
+		admin = nil
+	} else if err != nil {
+		return nil, err
+	}
+
+	ids, _, _, err := r.ClickhouseClient.QuerySessionIds(ctx, admin, projectID, 1_000_000_000, query, "ID", nil, retentionDate)
```

Comment:
Definite edge case, but wonder if we should limit the count to less than 1,000,000,000? I think clickhouse will only return a max of 1,000,000 rows before erroring and not sure if most programs can open more than a few hundred thousand rows

Reply:
good point... perhaps we can just error if there are more results? on the backend, it's kind of an edge case since the data is aggregated by user via the postgres query, so the large number of ids is just possible in between (with too many id arguments, the postgres query won't run anyway because of the argument limits). will make this 1M and throw an error if more results are returned by the query.